### PR TITLE
[DEP-188]: Deployment of initial release v1.0.0

### PR DESCRIPTION
**Pull request by rgcriado**
### Description

Support the [v1.0.0](https://github.com/getyoti/yoti-face-capture-ios/releases/tag/v1.0.0) release:
- Update README.md
- Add license
- Add gitignore
- Add pull request template

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,85 @@
-# yoti-face-capture-ios
+# YotiFaceCapture
+
+YotiFaceCapture uses the front facing camera to perform face detection, then validates frames where a face has been detected according to Yoti's DeepFas specifications and finally, it returns a result that will either contain a cropped image containing a face or an error.
```
Comment (jlandaz) [inline README.md:3]: I think we need someone business side to phrase this better.

File: README.md [hunk 2]
```diff
@@ -1 +1,85 @@
...
+```
+
+### 2. Create FaceCaptureViewController 
+Fetch FaceCaptureViewController from framework and set delegate
+```swift
+let faceCaptureViewController = FaceCapture.faceCaptureViewController()
+faceCaptureViewController.delegate = self
+```
+
+### 3. Start camera feed and analysis process
```
Comment (jlandaz) [inline README.md:38]:
```suggestion
### 3. Start camera feed and analysis
```

File: README.md [hunk 3]
```diff
@@ -1 +1,85 @@
...
+Start the camera feed
+```swift
+faceCaptureViewController.startCamera()
+```
+Start the analysis process
```
Comment (jlandaz) [inline README.md:43]:
```suggestion
Start the analysis
```

File: README.md [hunk 4]
```diff
@@ -1 +1,85 @@
...
+        case .success(.validFrame(let information)):
+            break
+        case .failure(let error):
+            break
+    }
+}
+```
+
+### 4. Customize framework configuration
+Create a FaceCaptureConfiguration object and pass the configuration when calling start analyzing method
```
Comment (jlandaz) [inline README.md:80]:
```suggestion
Provide a FaceCaptureConfiguration instance when calling the startAnalyzing method
```

File: README.md (hunk 1 above)
Comment (AdrienVioletYoti) [reply README.md:3]: yep and not sure we want to mention DeepFas, it is an internal name

**Review by jlandaz — APPROVED** [review]

**Review by AdrienVioletYoti — APPROVED** [review]

**Pull request merged**